### ISSUE TITLE
Version 3.3.0 is not automatically updated

### ISSUE THREAD

**Issue by LaurentGrenet** [mrdokenny/Cookie-AutoDelete#731]
I got notification yesterday that Version 3.3.0 has been deployed, nevertheless, it is still the 3.2.0 that is proposed in Firefox.

Is there something missed in deployment to have 3.3.0 proposed and/or automatically updated ?

**Comment by ghost** [comment]
CAD has "[Reccomended](https://support.mozilla.org/en-US/kb/recommended-extensions-program)" badge on AMO - it must go through code review.

**Comment by kennethtran93** [comment]
When any browser has reviewed the extension and approves it, it will be pushed out automatically.

I have setup a GitHub Action to push new persons for review to Chrome and Firefox when I create a new release, so that most likely won't change. GitHub Releases will have the version first. Firefox usually gets the approval sooner than the rest. Chrome I just need to hit the submit for review button and Edge Chromium I need to do the whole process manually.

**Comment by kennethtran93** [comment]
As @gwarser said, I have enrolled in Firefox's Recommended Extensions process, which requires every version to be manually reviewed. 

![20200518_125019](https://user-images.githubusercontent.com/6724477/82253634-310cd700-9906-11ea-9c06-374a0d15cf9d.jpg)

Shouldn't take too much longer. It was at 18th queue when I submitted it.

I usually edit the Release Notes on GitHub Releases when it gets released.
